quize.css

```diff
@@ -0,0 +1,95 @@
+
+* {
+	font-family: sans-serif;
+	/*font-style: bold;*/
+}
+/*body {
+	padding: 10px;
+
+}*/
+
+.discription {
+	padding-left: 10px;
+	padding-right: 10px;
+}
+h1 {
+
+	text-align: center;
+}
+ h2{
+
+ 	text-align: left;
+ 	margin-left: 10px;
+ 	font-weight: 400;
+ 	margin-bottom: 10px;
+ 	margin-top: 10px;
+ }
+
+/*section 1*/
+
+.section1 {
+	border: 1px dotted lightgray;
+}
```

Comment:
You put this on the section1, but it actually needs to go around your header for each card.